### DISCUSSION TITLE
Temp files

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#76] [Q&A] [answered]
Hello !

I'm wondering if there is anyway to change de folder where temp files are droped ?
I change for Blender but while I render, files from the addon continue to goes in the ancient location !

**Comment by pragma37** [comment, marked as answer]
Hi! I guess you mean the Render Output directory?

That changes the render directory only for new scenes, not even the default scene for new files.
For existing scenes, you'll have to change the output path in the scene settings panel.

This is directly handled by Blender. You would see the same behavior from EEVEE or Cycles.

**Reply by Airioo** [reply]
Hi !  Thanks for replying !

No I mean temps files that are created while I render : 

The render output is correctly go to the good location, but there are some files created (Idk what they are) in my c disk on a folder named "temp" !

**Reply by pragma37** [reply]
What kind of files? Name? Extension?

The only files that Malt writes to your disk are log files, and they always go to your OS true temporary folder (*C:\Users\<YourUserName>\AppData\Local\Temp* for Windows).

Anything else is written by Blender itself.

**Reply by Airioo** [reply]
Yes ! this is them ! But it tooks a lot of space, I would change the location if it's possible !

**Reply by pragma37** [reply]
I see.

The point of writing the logs to the Temporary Files folder is that Windows will automatically delete them from time to time.
If you move them to another folder that won't happen and they will keep taking your disk space unless you manually delete them.

I could simply add an option to disable logs, though.

How much space do they take? It shouldn't be that much.
Could you attach one of the logs?

**Reply by Airioo** [reply]
Hmmm My animation took something like 30 go :/

**Reply by pragma37** [reply]
30 GB ?!?!?!

That definitely should not happen.
Please render 2 frames and send me the log file so I can figure out what's going on.

**Reply by Airioo** [reply]
Log file of blender ?

**Reply by Airioo** [reply]
Ok so I found where it come from, it's while you check the "debug mod", thanks !
